libraries/ag-charts-test/src/canvas/conicGradient.ts

```diff
@@ -0,0 +1,94 @@
+import { Canvas, type CanvasRenderingContext2D as NodeCanvasRenderingContext2D } from 'canvas';
+
+interface ColorStop {
+    offset: number;
+    color: string;
+}
+
+function parseColor(input: string): number[] {
+    if (input.startsWith('#')) {
+        const parts: number[] = [];
+
+        const delta = input.length < 6 ? 1 : 2;
+        for (let i = 1; i < input.length; i += delta) {
+            const part = input.slice(i, i + delta);
+            const value = parseInt(part, 16);
+            if (delta === 1) {
+                parts.push(value + value * 16);
+            } else {
+                parts.push(value);
+            }
+        }
+
+        return parts;
+    } else if (input.startsWith('rgb')) {
+        const rgba = Array.from(input.matchAll(/[\d.]+/g), (match) => Number(match[0]));
+
+        if (rgba.length === 3) {
+            return [rgba[0], rgba[1], rgba[2]];
+        } else if (rgba.length === 4) {
+            return [rgba[0], rgba[1], rgba[2], rgba[3]];
+        }
+    }
+
+    throw new Error(`Failed to parse "${input}"`);
```

Comment:
We may want to also include the 140 named colour:
https://www.w3schools.com/cssref/css_colors.php

Reply:
Possibly. I've copied this from color.ts, since I can't import that file. I only really need to handle what outputs `color.ts` generates